dotrix_core/src/assets/shader.rs

```diff
@@ -16,6 +16,7 @@ impl Shader {
     /// Loads the shader to GPU
     pub fn load(&mut self, renderer: &Renderer) {
         if !self.module.loaded() {
+            self.module.label = self.name.clone();
```

Comment:
Without this the a shader error will report the module as "NoName" which makes debugging harder